### ISSUE TITLE
Resnet50 CUB 2011 performance

### ISSUE THREAD

**Issue by jeong-tae** [baidu-research/DT-RAM#2]
Hi,
I am looking for the best performance on CUB 2011 and i find that you've got the 84.5% accuracy with resnet50. You mentioned that you've done fine-tuning very carefully and got this result.

I also try to reproduce this performance with resnet50 on Pytorch framework. but doesn't work.

Here is my setting and let me know what kind of setting did you use. there is large difference you've got.
**Training argumentation:**
Image rescale size: 448 (keep aspect ratio)
image random crop(448 size)
random Hflip
normalization

batch size 16
initial lr: 1e-3
lr decay(*0.1) at 30, 60 epoch

**Test argumentation**
Image rescale size: 448 (keep aspect ratio)
image center crop(448 size)
normalization

I employ pretrained resnet50 from imagenet that provided by Pytorch community.
So far, i got about 83% accuracy on resnet152. with resnet50, still less than 80%

**Comment by Lzc6996** [comment]
@jeong-tae 
you can just use the [fb.resnet.torch](https://github.com/facebook/fb.resnet.torch) to reproduce the results of resnet50.

**Comment by jeong-tae** [comment]
@Lzc6996 Okay, I will try that and let you know results

**Comment by GuoleiSun** [comment]
@jeong-tae 
I have the same problem reproducing the results of resnet50 (84.5). I also got less than 80% using resent50.
@Lzc6996 Can you be more specific when you refer us to fb.resnet.torch. Basically, what argumentation do you use, how much is the learning rate, etc?
Thanks a lot.

**Comment by Lzc6996** [comment]
@GuoleiSun 
Did you use the pretrained model provided by [fb.resnet.torch](https://github.com/facebook/fb.resnet.torch/tree/master/pretrained)

**Comment by pgruening** [comment]
@jeong-tae 
Hello,
did using the torch version of the network improve your result? How did you convert the torch model to pytorch?

Thank you.

**Comment by jbcdnr** [comment]
Can you provide the hyperparameters of the training of the ResNet50? Thank you.
